app-misc/geoclue/geoclue-2.4.7.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
```

Comment:
Please bump to EAPI=6, in accordance with https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies#EAPI